Resources/Private/Language/locallang_be.xlf

```diff
@@ -115,6 +115,12 @@
 			<trans-unit id="newContentWizard.latest.description">
 				<source>Shows a configurable amount of newest news, optionally filtered by category or subcategory.</source>
 			</trans-unit>
+			<trans-unit id="layout.button.save">
```

Comment:
Not sure about the `layout.button.save` spelling vs `layout_edit_mode` — what's the reason for using `.` or `_`?

Reply:
Now using dots for consistencies sake